style.css

```diff
@@ -0,0 +1,5 @@
+#img-div img{
+    text-align: center;
+    height: 200px;
+     width: 200px;
```

Comment:
Please use proper indentation for your stylesheet in order to enhance neatness and presentability of your code.
Linters are a great tool to enable you achieve this. Kindly view this page to understand linters and how to work with them to enhance your code https://github.com/microverseinc/linters-config/tree/master/html-css